README.md

```diff
@@ -56,105 +39,25 @@ This list contains workflows covered by packit tool and links to the documentati
 - [Create a SRPM from the current content in the upstream repository.](https://packit.dev/docs/cli/srpm/)
 - [Sync content of the Fedora dist-git repo into the upstream repository.](https://packit.dev/docs/cli/sync-from-downstream/)
 
-## Configuration
-
-Configuration file for packit is described [here](http://packit.dev/docs/configuration/).
-
-TL;DR
-
-```yaml
-specfile_path: packit.spec
-synced_files:
-  - packit.spec
-upstream_package_name: packitos
-downstream_package_name: packit
-```
-
-## User configuration file
-
-User configuration file for packit is described [here](http://packit.dev/docs/configuration/#user-configuration-file).
-
 ## Requirements
 
 Packit is written in python 3 and is supported only on 3.6 and later.
 
-When packit interacts with dist-git, it uses `fedpkg`, we suggest installing it:
```

Comment:
We either need to remove the below `sudo dnf install -y fedpkg` as well (which I'd prefer) or keep this sentence.

Reply:
Ok, it's fine to know that it's what you expected. :+1:

Reply:
Not sure we understand each other :-)
I've been suggesting to remove `sudo dnf install -y fedpkg` as well. (I still see it there in `## Installation`)